config/environments/development.rb

```diff
@@ -34,7 +34,18 @@
   config.active_storage.service = :local
 
   # Don't care if the mailer can't send.
-  config.action_mailer.raise_delivery_errors = false
+  config.action_mailer.raise_delivery_errors = true
+  config.action_mailer.delivery_method = :smtp
+  config.action_mailer.smtp_settings = {
+    # EDITOR="vi" rails credentials:edit
+    user_name: 'service.eagle12@gmail.com',
```

Comment:
please do not push any credential in git.

you can use credential.yml for this